metricflow/plan_conversion/time_spine.py

```diff
@@ -34,26 +33,42 @@ def spine_table(self) -> SqlTable:
         return SqlTable(schema_name=self.schema_name, table_name=self.table_name, db_name=self.db_name)
 
     @staticmethod
-    def create_from_manifest(semantic_manifest: SemanticManifest) -> TimeSpineSource:
+    def create_from_manifest(semantic_manifest: SemanticManifest) -> Dict[TimeGranularity, TimeSpineSource]:
         """Creates a time spine source based on what's in the manifest."""
-        time_spine_table_configurations = semantic_manifest.project_configuration.time_spine_table_configurations
-
-        if not (
-            len(time_spine_table_configurations) == 1
-            and time_spine_table_configurations[0].grain == DEFAULT_TIME_GRANULARITY
-        ):
-            raise NotImplementedError(
-                f"Only a single time spine table configuration with {DEFAULT_TIME_GRANULARITY} is currently "
-                f"supported. Got:\n"
-                f"{mf_pformat(time_spine_table_configurations)}"
+        time_spine_sources = {
+            time_spine.primary_column.time_granularity: TimeSpineSource(
+                schema_name=time_spine.node_relation.schema_name,
+                table_name=time_spine.node_relation.relation_name,  # is relation name the table name? double check
```

Comment:
relation_name should be the table-or-view name.

Reply:
Whoops I forgot about this comment 😅 thank you!